desktop/src/main/java/bisq/desktop/main/offer/MutableOfferDataModel.java

```diff
@@ -424,7 +417,7 @@ void onCurrencySelected(TradeCurrency tradeCurrency) {
         }
     }
 
-    @Override
+    //@Override
     public void onUpdateBalances(Coin availableConfirmedBalance,
```

Comment:
remove the method please, it was implementing the method from the removed interface BsqBalanceListener